library/src/scala/tasty/reflect/Types.scala

```diff
@@ -292,7 +292,7 @@ trait Types {
   /** Pattern representing `X | Y | ...` alternatives. */
   type Alternatives <: Tree
 
-  /** A type */
+  /** A type, kind, type bounds or NoPrefix */
   type Type
```

Comment:
It is not clear what does `kind` refer to?

Reply:
Maybe I should just leave it as type an assume that `F` is equivalent to `[X1,...,Xn] => F[X1,...,Xn]`.

Reply:
I see, maybe _type constructor_ is more clear.